lib/Hoare_Sep_Tactics/Hoare_Sep_Tactics.thy

```diff
@@ -344,8 +344,7 @@ done
 
 
 ML \<open>
-   fun J f x = f x
-               handle _ => x   (* FIXME! exceptions *)
+   fun J f x = \<^try>\<open>f x catch _ => x\<close>   (* FIXME! exceptions *)
```

Comment:
Hmm, this FIXME could possibly be removed now that we're using the `try` antiquotation.

Reply:
Yes, let's do that.